tools/topology/topology2/include/pipelines/cavs/dai-copier-eqiir-gain-module-copier-capture.conf

```diff
@@ -0,0 +1,204 @@
+#
+# BE capture pipeline: dai-copier-eqiir-gain-module-copier-capture
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.dai-copier-eqiir-gain-module-copier-capture.N.attribute_name"
+#
+# Usage: dai-copier-eqiir-gain-module-copier-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.dai-copier-eqiir-gain-module-copier-capture."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/input_audio_format.conf>
+<include/common/output_audio_format.conf>
+<include/components/dai-copier.conf>
+<include/controls/bytes.conf>
+<include/components/eqiir.conf>
+<include/components/pipeline.conf>
```

Comment:
Include of gain.conf is missing, build it worked because it was included somewhere else.